packages/docs/versioned_docs/version-2.0.0-alpha-3/welcome.md

```diff
@@ -8,21 +8,27 @@ title: What is UniRep?
 
 ## Overview
 
-**UniRep** (**Uni**versal **Rep**utation) is a _private_ and _non-repudiable_ **reputation system**. Users can&#x20;
+**UniRep** (**Uni**versal **Rep**utation) is a _private_ and _non-repudiable_ **reputation system**. Users can:&#x20;
 
 1. Receive positive and negative reputation from attesters.
-2. Voluntarily prove that they have at least certain amount of reputation without revealing the exact amount.&#x20;
-3. Moreover, users cannot refuse to receive reputation from an attester.
+2. Voluntarily prove that they have at least a certain amount of reputation without revealing the exact amount.&#x20;
+3. Users cannot refuse to receive reputation from an attester.
 
-The high-level goal for **UniRep** is to be a base layer on top of which anyone can easily build _custom_, yet _interoperable_, reputation systems, For instance, users could create combined zero-knowledge proofs of reputation across different social media platforms, consumer apps, or financial applications, in order to provide holistic, private, and trustworthy information about themselves to others.
+The high-level goal for **UniRep** is to be a base layer on top of which anyone can easily build _custom_, yet _interoperable_, reputation systems. For instance, users could create combined zero-knowledge proofs of reputation across different social media platforms, consumer apps, or financial applications in order to provide holistic, private, and trustworthy information about themselves to others.
+
+TODO: link to new Medium article from Althea and Ki Chong [here]
```

Comment:
make sure to open issues for TODO sections that we leave in. Don't want to forget to update

Reply:
✅